web/app/ratings_helpers.py

```diff
@@ -0,0 +1,22 @@
+
+
+def rating_to_traditional_range(rating_f):
```

Comment:
Does "traditional range" actually exist? My impression of AGA raw ratings was that all positive ratings were "rounded up" so that [0, 1] = 1dan, whereas negative ratings were rounded down so that [-1, 0] = 1kyu.